### PR TITLE
Update mers v3

### PR DESCRIPTION
**Pull request by luisfponce**
Do Not Merge until explicitly required please

### CODE REVIEW

File: mers/releasenotes.md [hunk 1]
```diff
@@ -31,21 +30,15 @@ surveillance, and product placement.
...
-     the version that comes by default with the docker Media Stack image. The
-     last validate version for the stack is `31410`.
+  * Ubuntu* OS `20.04 LTS (Focal Fossa)`
+  * x264 version `1771b556ee45207f8711744ccbd5d42a3949b14c`
+  * SVT-HEVC `ead6fdf7c9ff84511b42fc1658c1654b84d83e4b`
+  * SVT-AV1 `v0.8.4`
+  * FFmpeg `7800cc6e82068c6dfb5af53817f03dfda794c568`
+  * GStreamer `1.18.0`
+  * OPENCV `4.4.0`
+  * OpenVINO (DLDT) `2020.4`
```
Comment (DnPlas) [inline mers/releasenotes.md:40]: I think you have to specify which component, in this case the model server.

File: mers/ubuntu/ocl/sysstacks-mers-ubuntu-v3-include-ocl.diff [hunk 2]
```diff
@@ -0,0 +1,31 @@
+diff --git a/Dockerfile b/Dockerfile
```
Comment (DnPlas) [inline mers/ubuntu/ocl/sysstacks-mers-ubuntu-v3-include-ocl.diff:1]: Why including a patch of the dockerfile?

File: mers/ubuntu/aom-patches/stacks_mers-v3-include-aom.diff [hunk 3]
```diff
@@ -0,0 +1,42 @@
+diff --git a/Dockerfile b/Dockerfile
+index 33cba21..100e19c 100644
+--- a/Dockerfile
++++ b/Dockerfile
```
Comment (DnPlas) [inline mers/ubuntu/aom-patches/stacks_mers-v3-include-aom.diff:4]: Same as below, why including patches of a dockerfile?

File: mers/releasenotes.md [hunk 4]
```diff
@@ -79,7 +79,8 @@ the corresponding licenses
 |OpenCV	             |BSD 3-clause
 |OpenVINO	         |Apache License v2
 |gst-video-analytics |MIT
-|ClearLinux	         |Collection of license - https://download.clearlinux.org/current/licenses
+|Clear Linux* OS	         |Multiple
```
Comment (DnPlas) [inline mers/releasenotes.md]: If this is deprecated, let's not list it

File: mers/NEWS.md [hunk 5]
```diff
@@ -31,11 +76,23 @@ This release enables execution on Intel® Processor Graphics.
...
+    * GST PLUGINS GOOD -> `1.16.0`
+    * GST PLUGINS BAD -> `1.16.0`
+    * GST PLUGINS UGLY -> `1.16.0`
   - OPENCV ->  `4.1.2`
   - DLDT ->  `2019_R3.1`
   - VA_GSTREAMER_PLUGINS ->  `0.6.1`
 
+- Intel highlights:
+  - Enable Intel® IPP (ICV version) 2019.0.0 on OpenCV
+  - Enable Intel® Math Kernel Library for Deep Neural Networks (Intel® MKL-DNN) `v1.0.1` on DLDT
```
Comment (DnPlas) [inline mers/NEWS.md]: This changed names to onednn, please add the correct format

**Review by DnPlas — CHANGES_REQUESTED** [review]

File: mers/releasenotes.md (hunk 1 above)
Comment (luisfponce) [reply mers/releasenotes.md:40]: We are, and in this case MeRS is not using OVMS, we are using DLDT.

File: mers/ubuntu/aom-patches/stacks_mers-v3-include-aom.diff (hunk 3 above)
Comment (luisfponce) [reply mers/ubuntu/aom-patches/stacks_mers-v3-include-aom.diff:4]: This patches are required for the blog.

The blogs guide interested people to build its own MeRS, and also to build it to get couple of components not included due legal reasons or due different interest of the stack.

Quick background:
* Patch to AOM, component not releasable within the stack due legal reasons.
* Patch to OCL,  component not released due internal discussions.

File: mers/ubuntu/ocl/sysstacks-mers-ubuntu-v3-include-ocl.diff (hunk 2 above)
Comment (luisfponce) [reply mers/ubuntu/ocl/sysstacks-mers-ubuntu-v3-include-ocl.diff:1]: Same as the other comment.

File: mers/releasenotes.md (hunk 1 above)
Comment (DnPlas) [reply mers/releasenotes.md:40]: Gotcha, shouldn't it be Openvino toolkit?

File: mers/NEWS.md (hunk 5 above)
Comment (luisfponce) [reply mers/NEWS.md]: You are ryt

File: mers/releasenotes.md (hunk 1 above)
Comment (luisfponce) [reply mers/releasenotes.md:40]: Yes, a.k.a. _Deep Learning Deployment Toolkit_ or DLDT

File: mers/releasenotes.md (hunk 4 above)
Comment (luisfponce) [reply mers/releasenotes.md]: Done

File: mers/NEWS.md (hunk 5 above)
Comment (luisfponce) [reply mers/NEWS.md]: Done

**Review by DnPlas — APPROVED** [review]

**Comment by DnPlas** [comment]
LGTM, thanks @luisfponce

**Pull request merged**